src/ServiceControl.Audit/Auditing/AuditPersister.cs

```diff
@@ -301,16 +283,12 @@ await messageDispatcher.Value.Dispatch(new TransportOperations(messagesToEmit.To
             }
         }
 
-        readonly Counter ingestedAuditMeter;
-        readonly Counter ingestedSagaAuditMeter;
-        readonly Meter auditBulkInsertDurationMeter;
-        readonly Meter sagaAuditBulkInsertDurationMeter;
-        readonly Meter bulkInsertCommitDurationMeter;
-        readonly IMessageSession messageSession;
-        readonly Lazy<IMessageDispatcher> messageDispatcher;
+        readonly Counter<long> storedAudits = AuditMetrics.Meter.CreateCounter<long>($"{AuditMetrics.Prefix}.stored_audit_messages");
```

Comment:
Same here, these prefixes shouldn't be used and isn't according to spec. The `Meter` should define the prefix/scope AFAIK:

- https://learn.microsoft.com/en-us/dotnet/core/diagnostics/metrics-instrumentation

```c#
var meter = meterFactory.Create("Particular.ServiceControl.Audit");
_hatsSold = meter.CreateCounter<int>("stored_messages_total");
```

or

```c#
var meter = meterFactory.Create("Particular.ServiceControl");
_hatsSold = meter.CreateCounter<int>("audit.stored_messages_total");
```

but as we have seperate instance my preference would be the 1st.

Reply:
Fixed